hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmSnapshotManager.java

```diff
@@ -43,12 +49,16 @@
 import org.apache.hadoop.ozone.snapshot.SnapshotDiffReport;
 import org.apache.hadoop.ozone.snapshot.SnapshotDiffResponse;
 import org.apache.ozone.rocksdiff.RocksDBCheckpointDiffer;
+import org.rocksdb.ColumnFamilyDescriptor;
+import org.rocksdb.ColumnFamilyHandle;
```

Comment:
I thought those classes were banned for direct use. @duongkame put it in the allow list: https://github.com/apache/ozone/blob/354d3dc5a16eade52bdcbd566bfb952ac3a1d8b6/pom.xml#L1760-L1774

Should be good.